tests/methodologies/para/test_ai_heuristic.py

```
@@ -62,7 +62,12 @@ def test_ollama_not_installed(self, tmp_path: Path) -> None:
         test_file = tmp_path / "notes.txt"
         test_file.write_text("project deadline tomorrow")
 
-        with patch(f"{_HEURISTICS_MODULE}.OLLAMA_AVAILABLE", False):
+        # Patch both OLLAMA_AVAILABLE and ollama to prevent any real or leaked
+        # mock client from reaching _parse_response (xdist isolation defence).
+        with (
+            patch(f"{_HEURISTICS_MODULE}.OLLAMA_AVAILABLE", False),
```

Ruff (0.15.10)

[warning] Boolean positional value in function call

(FBT003)